src/components/PersonRow/PersonRow.tsx

```diff
@@ -0,0 +1,40 @@
+import { Person } from '../../types/Person';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person;
+  people: Person[];
+  slug?: string;
+};
+
+export const PersonRow: React.FC<Props> = ({ person, slug, people }) => {
+  return (
+    <tr
+      data-cy="person"
+      key={person.slug}
+      className={person.slug === slug ? 'has-background-warning' : ''}
```

Comment:
Use the classnames library rules for add classes with condition, fix it everywhere